doc/doc-site/docs/about/how-it-works.md

```diff
@@ -43,28 +43,35 @@ blockchain.
 
 ### First steps: launch your emojicoin
 
-Launching an emojicoin costs just 1 APT! Note the following:
+Launching an emojicoin costs just 1 APT plus a 4 APT refundable deposit! Note
+the following:
 
 - Single emojis and emoji combinations are supported, as long as the total
   number of bytes is 10 or less.
 - Only one market may be launched for a given emoji combination (emojicoin
   markets are canonical).
 - Emojis that became part of the [Unicode emoji library] after the launch of
   emojicoin.fun are not supported.
+- The 4 APT refundable deposit will be sent back to the market registrant once
+  the market exits the bonding curve.
+- After the market is registered, a 5 minute grace period begins, during which
+  only the market registrant may place the first swap. After the first swap or
+  5 minutes, whichever comes first, anyone can swap!
 
 ### Growing up: exiting the bonding curve
 
 As users buy or sell against the bonding curve, the market capitalization of the
-emojicoin changes. Once 10,000 APT of cumulative sell pressure has pushed the
-bonding curve up to a market capitalization up to 45,000 APT, the emojicoin
-leaves the bonding curve.
+emojicoin changes. Once 1,000 APT of cumulative buy pressure has pushed the
+bonding curve up to a market capitalization up to 4,500 APT, the emojicoin
+leaves the bonding curve and the 4 APT deposit is automatically sent back to the
+market registrant.
```

Comment:
```suggestion
emojicoin changes. Once 1,000 APT of cumulative buy pressure has pushed the
bonding curve up to a market capitalization of 4,500 APT, the emojicoin
leaves the bonding curve and the 4 APT deposit is automatically sent back to the
market registrant.
```
```suggestion
emojicoin changes. Once 1,000 APT of cumulative buy pressure has pushed the
bonding curve up to a market capitalization up to 4,500 APT, the emojicoin
leaves the bonding curve and the 4 APT deposit is automatically sent back to the
market registrant.
```

Reply:
Or maybe you meant `of up to`? Maybe it's just me but it reads a little awkward 😛

Reply:
Addressed in 9e7c0d1ad7ea62121f407c8da306b1dbb1baf22c